docs/_docs/04_reference/cli-options.md

```diff
@@ -71,7 +71,7 @@ This page documents the CLI commands and options available with Terragrunt:
   - [terragrunt-modules-that-include](#terragrunt-modules-that-include)
   - [terragrunt-fetch-dependency-output-from-state](#terragrunt-fetch-dependency-output-from-state)
   - [terragrunt-use-partial-parse-config-cache](#terragrunt-use-partial-parse-config-cache)
-  - [terragrunt-include-module-prefix](#terragrunt-include-module-prefix)
+  - [terragrunt-include-module-prefix (DEPRECATED: use terragrunt-raw-module-output)](#terragrunt-include-module-prefix)
```

Comment:
NIT: I think this is better like the following:

- [terragrunt-include-module-prefix](#terragrunt-include-module-prefix) (DEPRECATED: use [terragrunt-raw-module-output](#terragrunt-raw-module-output))

With the updated flag name.

Not a huge deal, though.

Reply:
Fixed.